tests/cli/grass/test_data/README.md

```diff
@@ -0,0 +1 @@
+(empty)
```

Comment:
Use .gitkeep for keeping the empty directory

https://stackoverflow.com/questions/7229885/what-are-the-differences-between-gitignore-and-gitkeep

Reply:
actually, I just want to mock a file here

Reply:
will remove it after the training test is integrated